getting_started/first_3d_game/03.player_movement_code.rst

```diff
@@ -183,6 +183,7 @@ fall speed separately. Be sure to go back one tab so the lines are inside the
 
         # Moving the Character
         velocity = target_velocity
+        position += delta * velocity
```

Comment:
This is not correct, this is what the `move_and_slide` method does, you shouldn't do this manually

Reply:
i believe there is some confusion
i have downloaded the most recent version of godot(4.2) and move_and_slide doesn't do that
i have attached a file for clarity, could you let me know if i am accidentally using an earlier version?

https://github.com/godotengine/godot-docs/assets/139103754/f94632a0-f00a-499e-9231-72c091ef3d83

Reply:
I can't tell from just this video, however I can tell that your code isn't exact and you haven't copied the code exactly from the demo, I'd suggest testing the official demo and seeing if it works, and if you've made any errors copying the code:
https://github.com/godotengine/godot-demo-projects/tree/master/3d/squash_the_creeps